t/nqp/108-vmhash.t

```diff
@@ -234,3 +234,16 @@ todo('Exceptions for iterators NYI on js', 2)
     if $backend eq 'js';
 ok($msg ne "", 'iterator throws after end');
 is($msg, 'Iteration past end of iterator', 'iterator throws correct exception after end');
+
+$msg := "";
+try {
+    my $h := nqp::hash("a", 42);
+    my $i := nqp::iterator($h);
+    nqp::deletekey($h, "a");
+    nqp::shift($i);
+    CATCH {
+        nqp::getmessage($_);
+    }
+}
+
+ok($msg eq "", 'No SEGV on a misused hash iterator');
```

Comment:
I'm not seeing `$msg` being changed?  Is that just to create a valid `ok` test?  If that was the intent, you can also say:

`pass 'No SEGV on a misused hash iterator'`

Reply:
Oh, my bad as I did not add `$msg = ...` bit into CATCH, huh.
Also, given it is not directly an exception, but a SEGV is maybe a better option, let's see...

Reply:
There is apparently no `pass` in nqp, so just corrected the bit.

Reply:
Altai-man: It looks like the test actually checks for *both* no SEGV (by happening at all) and no exception (by leaving `$msg` unchanged).  Might be worth changing the test description to match that.